### DISCUSSION TITLE
Admin Command Block Shop Exchanging XP Levels for Bonus Claim Blocks

### DISCUSSION THREAD

**Discussion by TenacityKrikkit** [GriefPrevention/GriefPrevention#1173] [Q&A] [answered]
Hi! I apologize in advance if this is answered elsewhere, I tried my best to resolve my question on my own prior to posting. I am attempting to develop a command string that will allow a player to push a button at spawn that will activate command blocks to exchange XP levels for bonus claim blocks; however, regardless of the command string I use, I am unable to get a command block to grant a player bonus claim blocks unless I include the exact player name (and not a placeholder). For example, using `/acb @p 100` fails to give the nearest player bonus claim blocks. Does anyone have experience with this issue?

I would greatly appreciate any insight or advice. Thank you!

**Comment by RoboMWM** [comment, marked as answer]
plugins don't support that syntax, at least not without manually parsing for it.

**Reply by TenacityKrikkit** [reply]
Ok, thank for you the prompt response! I appreciate it. 

This may be a silly question, again, so I apologize in advance, but could someone suggest a way of manually parsing for a nearest player syntax or a placeholder tag for player that would work with the `/acb` command, please? 

Thanks again!

**Reply by RoboMWM** [reply]
Probably an addon would have to implement. Probably plugins out there that interpret that syntax and fill in the appropriate name too.

**Reply by Jikoo** [reply]
I believe [PsudoCommands](https://www.spigotmc.org/resources/psudocommands-add-the-target-selector-to-plugin-commands.56738/) is the only maintained plugin for syntax interpretation.

**Reply by TenacityKrikkit** [reply]
That worked, thank you!
